src/layout/Summary/SummaryContent.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import cn from 'classnames';
+
+import { EditButton } from 'src/layout/Summary/EditButton';
+import classes from 'src/layout/Summary/SummaryContent.module.css';
+import { getPlainTextFromNode } from 'src/utils/stringHelper';
+import type { ISummaryComponent } from 'src/layout/Summary/SummaryComponent';
+import type { LayoutNodeFromType } from 'src/utils/layout/hierarchy.types';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+export interface SummaryContentProps {
+  onChangeClick: () => void;
+  changeText: string | null;
+  label: React.ReactNode;
+  summaryNode: LayoutNodeFromType<'Summary'>;
+  targetNode: LayoutNode | any;
```

Comment:
Do we need `| any` here? That just removes typescript support for that property..

Reply:
I guess it's because the check for `'renderSummaryBoilerplate' in component` is not narrowing the LayoutNode down to just summary-renderable-components anymore. I think it's best to keep the code strict and check that either inside this React component or outside it (and pass in the narrowed type).

Reply:
Good point, thanks Ole Martin :)